rln/src/protocol.rs

```diff
@@ -53,6 +61,97 @@ pub struct RLNProofValues {
     rln_identifier: Field,
 }
 
+pub fn serialize_witness(rln_witness: &RLNWitnessInput) -> Vec<u8> {
```

Comment:
Is this equivalent to https://github.com/kilic/rln? In terms of field order etc and usage through nwaku

Reply:
This is custom data structure, just the concatenation (LE by default) of the field values making a witness: 
```
identity_secret: Field,
path_elements: Vec<Field>,
identity_path_index: Vec<u8>,
x: Field,
epoch: Field,
rln_identifier: Field
```

Do you mean https://github.com/status-im/nwaku/blob/6c8ab0ab0f76fe765d95ef2f7bfb88d691a18bf7/waku/v2/protocol/waku_rln_relay/rln.nim#L48 ?